### PR TITLE
refactor(v2): improved Farsi default translations

### PR DESCRIPTION
**Pull request by massoudmaboudi**
## Motivation

Improved default Farsi translations for the classic theme.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Run yarn run start:v2 and check that the page http://localhost:3000/ is rendered properly.

## Related PRs

No

### CODE REVIEW

File: packages/docusaurus-theme-classic/codeTranslations/fa.json [hunk 1]
```diff
@@ -1,56 +1,110 @@
 {
   "theme.AnnouncementBar.closeButtonAriaLabel": "بستن",
+  "theme.AnnouncementBar.closeButtonAriaLabel___DESCRIPTION": "The ARIA label for close button of announcement bar",
```
Comment (slorber) [inline packages/docusaurus-theme-classic/codeTranslations/fa.json:3]: Can you remove those `___DESCRIPTION` attributes please? They should not be added

Comment (massoudmaboudi) [reply packages/docusaurus-theme-classic/codeTranslations/fa.json]: Done. Thanks

File: packages/docusaurus-theme-classic/codeTranslations/fa.json [hunk 2]
```diff
@@ -1,56 +1,56 @@
...
+  "theme.docs.versions.latestVersionSuggestionLabel": "برای دیدن آخرین نسخه این متن، نسخه {latestVersionLink} ({versionLabel}) را ببینید.",
+  "theme.docs.versions.unmaintainedVersionLabel": "نسخه {siteTitle} {versionLabel} دیگر به روزرسانی نمی شود.",
   "theme.docs.versions.unreleasedVersionLabel": "نسخه {siteTitle} {versionLabel} هنوز به صورت کامل منتشر نشده است.",
   "theme.lastUpdated.atDate": " در تاریخ {date}",
   "theme.lastUpdated.byUser": " توسط {user}",
-  "theme.lastUpdated.lastUpdatedAtBy": "آخرین به روزرسانی{atDate}{byUser}",
-  "theme.navbar.mobileSidebarSecondaryMenu.backButtonLabel": "← Back to main menu",
-  "theme.tags.tagsListLabel": ":برچسب ها",
+  "theme.lastUpdated.lastUpdatedAtBy": "آخرین بروزرسانی در {atDate} توسط {byUser}",
+  "theme.navbar.mobileSidebarSecondaryMenu.backButtonLabel": "← بازگشت به منو اصلی",
```
Comment (slorber) [inline packages/docusaurus-theme-classic/codeTranslations/fa.json:52]: going to merge this now, but please double-check that you don't need to "flip" the arrow?

image

**Comment by slorber** [comment]
thanks 👍

**Pull request merged**